pyeudiw/trust/handler/x509.py

```diff
@@ -0,0 +1,91 @@
+import logging
+from typing import Union
+from pyeudiw.trust.handler.interface import TrustHandlerInterface
+from pyeudiw.trust.model.trust_source import TrustSourceData, TrustParameterData
+from pyeudiw.trust.handler.exceptions import InvalidTrustHandlerConfiguration
+from pyeudiw.x509.verify import (
+    verify_x509_attestation_chain, 
+    get_expiry_date_from_x5c, 
+    der_list_to_pem_list, 
+    pem_list_to_der_list, 
+    get_x509_info
+)
+
+logger = logging.getLogger(__name__)
+
+class X509Hanlder(TrustHandlerInterface):
+    """
+    X509Handler is a trust handler implementation that extracts trust material from x509 certificates.
+    """
+    def __init__(
+        self, 
+        client_id: str, 
+        relying_party_certificate_chains_by_ca: dict[str, Union[list[bytes], list[str]]],
+        private_keys: list[dict[str, str]],
+        client_id_scheme: str = "x509_san_dns",
```

Comment:
```suggestion
        client_id_scheme: str = "x509_san_uri",
```

since the default_client_id is the global client_id that uses an https URL